drivers/Makefile

```diff
@@ -19,7 +19,7 @@ obj-$(CONFIG_I2S) += i2s/
 obj-$(CONFIG_PWM) += pwm/
 obj-$(CONFIG_ADC) += adc/
 obj-$(CONFIG_NET_L2_ETHERNET) += ethernet/
-obj-$(CONFIG_SLIP) += slip/
+obj-y += net/
```

Comment:
We could also have
`obj-$(CONFIG_NETWORKING) += net/`
here.

Reply:
Done.